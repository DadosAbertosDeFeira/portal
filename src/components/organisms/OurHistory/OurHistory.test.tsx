import { render } from "@testing-library/react";

import { OurHistory, type OurHistoryProps } from ".";

describe("OurHistory", () => {
  const makeSut = ({ ...props }: Partial<OurHistoryProps>) => {
    return render(<OurHistory {...props} />);
  };

  it("renders correctly", () => {
    const { asFragment } = makeSut({});
    expect(asFragment).toMatchSnapshot();
  });
});
