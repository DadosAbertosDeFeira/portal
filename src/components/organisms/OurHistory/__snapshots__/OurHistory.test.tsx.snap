// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OurHistory renders correctly 1`] = `[Function]`;
